### PR TITLE
Update defaultValue and defaultValueExpression descriptions

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by andrewtelnov — APPROVED** [review]

**Review by tsv2013 — APPROVED** [review]

File: src/question.ts [hunk 1]
```diff
@@ -1204,8 +1204,13 @@ export class Question extends SurveyElement
     return "";
   }
   /**
-   * Set the default value to the question. It will be assign to the question on loading the survey from JSON or adding a question to the survey or on setting this property of the value is empty.
-   * Please note, this property is hidden for question without input, for example html question.
+   * A default value for the question. Ignored for question types that cannot have a [value](https://surveyjs.io/Documentation/Library?id=Question#value) (for example, HTML).
+   *
+   * The default value is used as a question value in the following cases:
+   *
+   * - While the survey is being loaded from JSON.
```
Comment (arminal) [inline src/question.ts:1211]:
```suggestion
   * - while the survey is being loaded from JSON;
```

File: src/question.ts [hunk 2]
```diff
@@ -1204,8 +1204,13 @@ export class Question extends SurveyElement
...
+   * - The question is just added to the survey and does not have an answer yet.
```
Comment (arminal) [inline src/question.ts:1212]:
```suggestion
   * - the question is just added to the survey and does not yet have an answer;
```

File: src/question.ts [hunk 3]
```diff
@@ -1204,8 +1204,13 @@ export class Question extends SurveyElement
...
+   * - The respondent left the answer empty.
```
Comment (arminal) [inline src/question.ts:1213]:
```suggestion
   * - the respondent left the answer empty.
```

File: src/question.ts [hunk 4]
```diff
@@ -1220,9 +1225,17 @@ export class Question extends SurveyElement
...
+   *
+   * This expression applies until the question [value](https://surveyjs.io/Documentation/Library?id=Question#value) is specified by an end user or programmatically.
+   *
+   * An expression can reference other questions as follows:
+   *
+   * - `{other_question_name}`
+   * - `{panel.other_question_name}` (to access questions inside the same dynamic panel)
+   * - `{row.other_question_name}` (to access questions inside the same dynamic matrix or multi-column dropdown)
+   *
+   * An expression can also include built-in and custom functions for advanced calculations. For example, if the `defaultValue` should be today's date, set the `defaultValueExpression` to `"today()"`, and the corresponding built-in function will be executed each time the survey is loaded. Refer to the following help topic for more information: [Use Functions in Expressions](https://surveyjs.io/Documentation/Library#conditions-functions).
```
Comment (arminal) [inline src/question.ts:1238]:
```suggestion
   * An expression can also include built-in and custom functions for advanced calculations. For example, if the `defaultValue` should be today's date, set the `defaultValueExpression` to `"today()"` and the corresponding built-in function will be executed each time the survey is loaded. Refer to the following help topic for more information: [Use Functions in Expressions](https://surveyjs.io/Documentation/Library#conditions-functions).
```

**Review by arminal — COMMENTED** [review]
#GrammarReview

Short document. Review listing rules for our documentation.

**Pull request merged**